ENH Extend PDP for nominal categorical features (#18298)
scikit-learn/scikit-learn: doc/modules/partial_dependence.rst

You can access the newly created figure and Axes objects using ``plt.gcf()``
and ``plt.gca()``.

> If you wish to plot partial dependence of categorical features, you need to specify
> which features to be considered as such using the parameter `categorical_features`. This

Reviewer: ```suggestion
To make a partial dependence plot with categorical features, you need to specify
which features are categorical using the parameter `categorical_features`. This
```

> parameters takes a list of indices or names of the categorical features or a boolean

Reviewer: ```suggestion
parameter takes a list of indices, names of the categorical features or a boolean
```
